friends/src/components/Styled-Components.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
```

Comment:
Great job bringing in a tool from a previous section (_styled-components_) on this project Rob. This is a good habit to develop as it keeps the older material fresh, and you can start to develop intuition on when, and why you want to use certain tools 👍